### PR TITLE
enforce style and add aws test

### PR DESCRIPTION
**Pull request by daichengxin**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/diann_convert.py [hunk 1]
```diff
@@ -47,7 +52,8 @@ def convert(ctx, diann_report, exp_design):
     out_triqler = pd.DataFrame()
     out_triqler = out_msstats[['ProteinName', 'PeptideSequence', 'PrecursorCharge', 'Intensity', 'Run', 'Condition']]
     out_triqler.columns = ['proteins', 'peptide', 'charge', 'intensity', 'run', 'condition']
-    out_triqler.loc[:, "searchScore"] = 1 - report['PEP']
+    out_triqler.loc[:, "searchScore"] = report['Q.Value']
+    out_triqler.loc[:, "searchScore"] = -np.log(out_triqler["searchScore"])
```
Comment (jpfeuffer) [inline bin/diann_convert.py:56]: Put a comment with the repovwhere you got that from

File: bin/diann_convert.py [hunk 2]
```diff
@@ -31,6 +33,9 @@ def convert(ctx, diann_report, exp_design):
         s_header = data[empty_row + 1].replace("\n", "").split("\t")
         s_DataFrame = pd.DataFrame(s_table, columns=s_header)
 
+    # filter based on qvalue 0.01 so that downstream analysiss
```
Comment (jpfeuffer) [inline bin/diann_convert.py:36]:
```suggestion
    # filter based on qvalue parameter for downstream analysiss
```

File: modules/local/diannconvert/main.nf [hunk 3]
```diff
@@ -25,6 +25,7 @@ process DIANNCONVERT {
     diann_convert.py convert \\
         --diann_report ${report} \\
         --exp_design ${exp_design} \\
+        --qvalue_threshold 0.01 \\
```
Comment (jpfeuffer) [inline modules/local/diannconvert/main.nf:28]: Can you use the protein_level_fdr_cutoff for this?

Comment (ypriverol) [reply modules/local/diannconvert/main.nf:28]: Agreed.

**Review by ypriverol — APPROVED** [review]

File: bin/diann_convert.py [hunk 4]
```diff
@@ -47,7 +52,10 @@ def convert(ctx, diann_report, exp_design):
     out_triqler = pd.DataFrame()
     out_triqler = out_msstats[['ProteinName', 'PeptideSequence', 'PrecursorCharge', 'Intensity', 'Run', 'Condition']]
     out_triqler.columns = ['proteins', 'peptide', 'charge', 'intensity', 'run', 'condition']
-    out_triqler.loc[:, "searchScore"] = 1 - report['PEP']
+
+    # https://github.com/statisticalbiotechnology/dia_sum/blob/main/scripts/result_scripts/diann_output_to_triqler_output.py
```
Comment (jpfeuffer) [inline bin/diann_convert.py:56]: This does not apply anymore.

File: conf/test_full_dia.config [hunk 5]
```diff
@@ -0,0 +1,36 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running real full dia tests (DIA)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple test.
```
Comment (jpfeuffer) [inline conf/test_full_dia.config:5]: Not fast and simple

File: conf/test_full_lfq.config [hunk 6]
```diff
@@ -0,0 +1,34 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running real full-size tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple test.
```
Comment (jpfeuffer) [inline conf/test_full_lfq.config:5]: Not fast and simple

File: modules/local/diannsummary/main.nf [hunk 7]
```diff
@@ -41,6 +44,7 @@ process DIANNSUMMARY {
             --use-quant \\
             --matrices \\
             --out diann_report.tsv \\
+            --qvalue $params.psm_level_fdr_cutoff \\
```
Comment (jpfeuffer) [inline modules/local/diannsummary/main.nf:47]: I thought the final output are proteins?

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

**Comment by jpfeuffer** [comment]
@ypriverol I found some minor doc errors and wrong FDR parameter.

**Pull request merged**